docs/process/metrics.md

```diff
@@ -0,0 +1,51 @@
+
+You can always see our recently released user facing features and fixes in the [release notes](https://desktop.github.com/release-notes/) of our marketing page.
+
+## Dimensions
+These are general metrics about users that are aggregated to understand general user behavior and to help us segment usage patterns. These helps us prioritize work, predict technical and performance implications of our features, and plan for devOps needs. 
+
+| Metric | Description | Justification |
+|:--|:--|:--|
+| `version` | The version of Desktop. | To visualize update rates and performance metrics for each version of Desktop, so we can make sure our users are updating the app and understand our most used versions. |
+| `osVersion` | The OS version. | To understand the most common versions of Windows and MacOS people use Desktop on, so we can more accurately prioritize version-specific bugs. |
+| `platform` | The OS. | To understand which platforms are most popular among people who use Desktop, so we can more accurately prioritize platform-specific bugs.  |
+| `repositoryCount` | The total number of repositories. | To understand the typical number of repositories connected to Desktop, so we can make appropriate decisions in terms of performance and UI. |
+| `gitHubRepositoryCount` | The number of GitHub repositories. | To understand the typical number of repositories connected to Desktop that are hosted on GitHub, so we can more accurately prioritize GitHub workflows in Desktop. |
+| `guid` | The unique ID of a desktop installation. | This allows us to aggregate metrics across multiple days, so we can understand how many people use Desktop per week or per month, and how many people use a particular feature per month.  |
+| `dotComAccount` | Lets us know if a user logged in with a GitHub.com account. | To understand what percentage of people who use Desktop use it with github.com, so we can more accurately prioritize GitHub workflows in Desktop. |
+| `enterpriseAccount` | Lets us know if a user logged in with an Enterprise account. | To understand what percentage of people who use Desktop use it with an Enterprise instance of GitHub, so we can more accurately prioritize Enterprise-related bugs. |
+| `theme` | The name of the currently selected theme/application appearance as set at time of stats submission. | To understand usage patterns of the Dark Theme feature, so we can more accurately prioritize theme-related bugs. |
+| `usage` | Constant used to group/filterin data warehouse | Specifies that this data is related to GitHub Desktop usage, so we can filter it correctly in our analytics. |
+
+## Measures
+These are general metrics about general feature usage and specific feature behaviors. These help us understand our users' mental map of the application, hypothesize pain points within the application, and aid us in feature and bug fix planning so that we can improve workflows that might actually be used.
```

Comment:
:art: drop the first `general` in the first sentence

Reply:
> aid us in feature and bug fix planning so that we can improve workflows that might actually be used

Maybe

> aid in feature and bugfix planning so that we can improve workflows that are more likely to benefit users